android/src/main/java/com/doublesymmetry/kotlinaudio/players/components/MediaFactory.kt

```diff
@@ -34,6 +34,10 @@ class MediaFactory (
     private val cache: SimpleCache?
 ) : MediaSource.Factory {
 
+    companion object {
+        private const val DEFAULT_USER_AGENT = "react-native-track-player/5.0.0"
```

Comment:
I'm not sure about specifying the version here. I know this is conventional for userAgents, however needing to manually keep this up-to-date with each version release is not something I like the sound of.

Previously this was being [defaulted to `react-native-track-player`](https://github.com/doublesymmetry/KotlinAudio/blob/bf71120704bfe4be2311cf86fc1e2ee1c3c702b7/kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/BaseAudioPlayer.kt#L600C52-L600C65). I would say we either stick with that or we find a way to pull the version in dynamically.

Reply:
adjusted, thanks for input.